### ISSUE TITLE
Test working in Postman but fails in Newman

### ISSUE THREAD

**Issue by rakeshk1987** [postmanlabs/newman#1557]
1. Newman Version : **3.9.3**
 2. OS details : **Win 7 64 Bit**
 3. Are you using Newman as a : **CLI**
 3. Did you encounter this recently, or has this bug always been there: Recently
 4. Expected behaviour: The "**pm.expect(res).to.be.badRequest**" test is working fine in Postman but fails in Newman
 5. Command / script used to run Newman: 
> newman run "Reg_Test Suite.json" -e "DEV.json" -g "globals.json" -r html,cli,json,junit

**Newman error:** 
```
_expected response to have status reason 'BADREQUEST' but got 'BAD REQUEST'
at assertion:0 in test-script
inside "B-One Regression Testing / Neg: Polaris B-One Service for Invalid Header"_
```

**Postman Test**: _pm.expect(res).to.be.badRequest;_

**Comment by kunagpal** [comment]
@rakeshk1987 Could you confirm that this bug was fixed with Newman v3.9.4?

**Comment by rakeshk1987** [comment]
Yes. The bug is fixed in Newman v3.9.4.

Please go ahead and close the bug.
